### ISSUE TITLE
Did anyone can stabilize footage from Insta360 One-R 1" ?

### ISSUE THREAD

**Issue by Fabersa** [gyroflow/gyroflow#249]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you tried the latest build?

- [X] I have tried the latest build

### Gyroflow version

1.0.0 rc4

### What operating system are you using?

MacOS 12.1

### What GPU are you using?

AMD FirePro D700 6gb

### What happened?

Me and some friend are trying to stabilize drone footage shooted with the Insta360 One-R 1 inch camera without success. First we pointed the problems to the drone frame, then to the camera but neither of those seems the problem who generate the wobbly video output. Anyone had success with this camera and Gyroflow?

**Comment by AdrianEddy** [comment]
Yes, Insta360 One-R works fine, please post your file so we can check

**Comment by Fabersa** [comment]
Can anyone try to stabilize this sample video from my One-R to check if I'm doing wrong or somethings fails on my camera? It's 5.7k ultrawide. Thanks
https://drive.google.com/file/d/1_JNonLLmzi0erD9bBVh5E5HQz2supLCL/view?usp=sharing

**Comment by dronotron** [comment]
just calibrate your camera gyro before your flight

**Comment by AdrianEddy** [comment]
The file itself works fine in Gyroflow, but you have a lot of noise in the gyro data - it's a hardware issue, maybe you need to soft mount the camera or fine tune your frame. 
I also tried in Insta360 studio and it also can't handle these vibrations well, so there's nothing more Gyroflow can do here.
